### PR TITLE
fix: fix the background of comments

### PR DESCRIPTION
**Pull request by JivusAyrus**
#### Checklist

- [x] run `make test`
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by jensneuse — APPROVED** [review]

**Comment by JivusAyrus** [comment]
SORRY, STILL WORKING ON IT. THE CHANGES I MADE DID NOT WORK.
CONVERTING IT TOI DRAFT

**Comment by jensneuse** [comment]
> SORRY, STILL WORKING ON IT. THE CHANGES I MADE DID NOT WORK.
> CONVERTING IT TOI DRAFT

Cpt. Capslock, is it you? :-)

File: docs-website/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "browserslist": "defaults, not ie <= 11",
   "dependencies": {
     "@docsearch/react": "^3.1.1",
-    "@giscus/react": "^2.2.0",
+    "@giscus/react": "2.2.0",
```
Comment (StarpTech) [inline docs-website/package.json:15]: Is that something we should document?

**Review by StarpTech — COMMENTED** [review]
LGH

File: docs-website/package.json (hunk 1 above)
Comment (JivusAyrus) [reply docs-website/package.json:15]: Should I mention it in the README?

Comment (JivusAyrus) [reply docs-website/package.json:15]: Not sure if we need to.

Comment (StarpTech) [reply docs-website/package.json:15]: Up to you. I just thought it might be easy to break again but a fixed version should also have some meaning.

**Review by StarpTech — APPROVED** [review]
LGTM

**Review by Pagebakers — APPROVED** [review]
LG

**Pull request merged**